clickable.yaml

```diff
@@ -1,47 +1,40 @@
 clickable_minimum_required: '7'
 builder: go
+prebuild: mkdir -p ${BUILD_DIR}/axolotl-web && cp ${ROOT}/axolotl-web/dist/* ${BUILD_DIR}/axolotl-web -R
 postbuild: ${ROOT}/scripts/postbuild.sh ${INSTALL_DIR};
 install_bin:
   - ${CRAYFISH_LIB_INSTALL_DIR}/bin/crayfish
+install_root_data:
+  - manifest.json
+  - click/axolotl.png
+  - click/textsecure-helper
+  - click/textsecure-push-helper.json
+  - click/textsecure-push.apparmor
+  - click/textsecure.apparmor
+  - click/textsecure.content-hub
+  - click/textsecure.desktop
+  - click/textsecure.png
+  - click/textsecure.url-dispatcher
+  - ${BUILD_DIR}/axolotl-web
+  - guis/qml/ut
 dependencies_host:
   - gettext
-kill: textsecure
-ignore:
-  - .*
-  - .clickable
-  - .git
-  - .github
-  - .gitignore
-  - .gitmodules
-  - click
-  - crayfish
-  - docker
-  - main.go
-  - po
-  - scripts
-  - axolotl-web
-  - build
-  - output
-  - '*.snap'
-  - main
-  - bind*.go
-  - '*.syso'
-  - '*.go'
-  - dist
-  - .flatpak-builder
-  - .idea
-  - flatpak
-  - libzkgroup*.so
-  - \$GITHUB_WORKSPACE/*
+kill: axolotl
 libraries:
   crayfish:
     builder: rust
     src_dir: crayfish
     rust_channel: 1.55.0
-  nodejs_deps:
+    docker_image: rust
```

Comment:
In order to fix the CI, you'll need to configure the crayfish job to use the `rust` docker image and install clickable there via pip.

Reply:
But you will need to separate the jobs that build crayfish and the app.

Reply:
Here we would need an docker in docker approach ;) @olof-nord i have separated also the axolotl-web building from building the click package, so we can reuse the build artifacts from that step, too.

Reply:
Do we? Can't you just specify a different image for one of the github actions job?

Reply:
the error was introduced by `vergen` which isn't needed. So we can use the normal clickable containers